.travis.yml

```diff
@@ -39,6 +39,7 @@ job_compile_common: &job_compile_common
     - $PYTHON_EXECUTABLE -m pip install -r requirements.txt
     # Install the package in editable mode.
     - VERBOSE=1 $PYTHON_EXECUTABLE -m pip install -v -e ".[visualization]"
+    - VERBOSE=1 $PYTHON_EXECUTABLE -m pip install -v ".[bert]"
```

Comment:
can you remove this from travis again? we do not test the examples in travis. Thanks